Cargo.toml

```diff
@@ -21,6 +21,7 @@ opt-level     = "z"
 strip         = "none"
 
 [workspace.lints.rust]
+unexpected_cfgs = { level = "allow", check-cfg = ['cfg(used_linker)'] }
```

Comment:
after upgrade `rust` to latest `1.84.0`, run `just lint` it output: 
![image](https://github.com/user-attachments/assets/05e5dae0-c923-486e-9091-6b754509bc90)
I am not sure if it is best way to solve this feel free to correct me, @Brooooooklyn